src/components/styles/Typography.tsx

```diff
@@ -12,3 +12,9 @@ export const wordBreakCss = css`
 export const WordBreak = styled.span`
   ${wordBreakCss};
 `;
+
+export const TextEllipsis = styled.div`
```

Comment:
[nit] Would a `<span>` be more appropriate here? Seems like we may not want to create block element if necessary (could always wrap the span in a div if needed!)